mitodl/micromasters
# Added avatar API and frontend

#### What are the relevant tickets?

closes #179 

#### What's this PR do?

This allows the user to upload a new avatar. There is a little 'photo' icon on the profile picture, and you can click that to open a dialog that lets you pick and then crop a new avatar.

#### Where should the reviewer start?

I think read through the new code and the changes to the `ProfileImage` component.

NO TESTS YET!

#### How should this be manually tested?

You should be able to update your picture! Cropping should update it to the cropped area. I think it should be fine with any image format (although I've only tested `.jpg` and `.png`).

#### Any background context you want to provide?

#### Screenshots (if appropriate)

![drop](https://cloud.githubusercontent.com/assets/6207644/18724591/f59a2ff0-800a-11e6-9388-0371bc3d95b4.png)

![crop](https://cloud.githubusercontent.com/assets/6207644/18724595/f886fdb0-800a-11e6-98c7-23ea0c6e07fd.png)

![change](https://cloud.githubusercontent.com/assets/6207644/18724599/fbb2e878-800a-11e6-84a0-6698d070b858.png)

#### What GIF best describes this PR or how it makes you feel?
